### DISCUSSION TITLE
2022.10.08 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#112] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
## 이번 주 목표 및 달성률(2022.10.02 ~ 2022.10.08)
이은혁 : 자바의 정석 기초편 배열까지 읽고 실습진행
고영석 : 알고리즘 문제 3개 이상 풀기, 진도 따라가기, 과목평가 pass하기, 블로그 글 3개 이상 올리기
김지애 : 알고리즘 5개 풀기, 싸피 진도 따라가기
김지오 : IM대비 swea/백준 문제 매일 2개 이상 풀기, 복싱 주3회, 다음주 주간평가 pass 할정도로 공부
나혜승 : 알고리즘 1일 2문제, 수업진도 따라가기
이수연 : 생활코딩 자바 입문 수업 5강 이상 수강, 블로그 7개 이상 작성, 깃허브 주 5회 이상 커밋
이문삼 : 싸피 진도 따라가기, 자바스크립트 강의 15개 이상 듣기

**Comment by HyeseungNA** [comment]
목표: 알고리즘 1일 두문제씩 풀기, 수업진도 따라가기
점수 : 70점
이유: 매일은 아니더라도 총 문제 풀어야 할 횟수는 달성했음, 장고는 예전 수업내용이 잘 기억 나지 않아 수업 이해하기 힘들었음

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 자바의 정석 기초편 배열까지 읽고 실습진행
점수 : 30점
이유 : 아직 연산자까지 밖에 못함.. 계속 목표가 바뀌고 있어서 뭔가 하나를 끝까지 못내고 있음.. 빨리 확실한 플랜을 세워야할듯싶음

**Comment by moonthree** [comment]
목표 : 싸피 진도 따라가기, 자바스크립트 강의 15개 이상 듣기
점수 : 50점
이유 : 싸피 진도는 따라갔지만, 자바스크립트 강의는 10개도 못 들었음

**Comment by swjiae** [comment]
목표 : 알고리즘 5개 풀기, 싸피 진도 따라가기
점수 : 60점
이유 : 알고리즘 X, 싸피 진도 따라가기OX, CS 공부 시작함

**Comment by YoungSeok222222** [comment]
목표: 알고리즘 문제 3개 이상 풀기, 진도 따라가기, 과목평가 pass하기, 블로그 글 3개 이상 올리기
점수:  70점
이유: 알고리즘 문제 3개 이상 풀기(완료), 진도 따라가기(강의는 들었지만, 그렇다고 장고를 할 수 있는 정도는 아님), 블로그 글 3개 이상 올리기 (완료),

**Comment by sooyeonlee127** [comment]
목표: 생활코딩 자바 입문 수업 5강 이상 수강, 블로그 7개 이상 작성, 깃허브 주 5회 이상 커밋
점수: 15점
이유: 자바 입문 수업 2강 수강, 블로그 글 5개 작성,  깃허브 3회 커밋 했습니다 ㅠㅠ 슬럼프 시작 .. 정신을 차리도록 하겠습니다 ㅠ
